packaging/installer/netdata-updater.sh

```diff
@@ -44,6 +44,26 @@ error() {
   echo >&3 "$(date) : ERROR: " "${@}"
 }
 
+: "${ENVIRONMENT_FILE:=THIS_SHOULD_BE_REPLACED_BY_INSTALLER_SCRIPT}"
+
+if [ "${ENVIRONMENT_FILE}" == "THIS_SHOULD_BE_REPLACED_BY_INSTALLER_SCRIPT" ]; then
+  if [ -r "${script_dir}/../../../etc/netdata/.environment" ]; then
+    ENVIRONMENT_FILE="${script_dir}/../../../etc/netdata/.environment"
+  elif [ -r "/etc/netdata/.environment" ]; then
+    ENVIRONMENT_FILE="/etc/netdata/.environment"
+  elif [ -r "/opt/netdata/etc/netdata/.environment" ]; then
+    ENVIRONMENT_FILE="/opt/netdata/etc/netdata/.environment"
+  else
+    envpath="$(find / -type d \( -path /sys -o -path /proc -o -path /dev \) -prune -false -o -path '*netdata/.environment' -type f  2> /dev/null | head -n 1)"
```

Comment:
We should verify this works fine on MacOS and FreeBSD (ie. if we haven't already done so).

Reply:
It _should_ work both places as the whole line is POSIX compliant in both syntax and options for the tools, but I will make a point to double-check it prior to merging.

Reply:
Confirmed, it works on both macOS and FreeBSD.